source/_integrations/fyta.markdown

```diff
@@ -0,0 +1,54 @@
+---
+title: Fyta
+description: Instructions on how to integrate Fyta sensors within Home Assistant.
+ha_category:
+  - Sensor
+ha_release: 0.1
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@dontinelli'
+ha_domain: fyta
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+The `fyta` integration uses the open APY of the [FYTA](https://www.fyta.de) to obtain the data from your plant sensors and integrate these into your Home Assistant installation.
+
+For the authentication on the FYTA-server you need your login-credentials (email and password). At a later stage the login via access token generated in the FYTA portal may also be supported.
+
+The integration provides a device for the controller interface and all plants with a Fyta Beam sensor.
+
+{% include integrations/config_flow.md %}
+
+## Sensors
+
+The following sensors are currently available for the controller:
+
+| name                  | Unit   | Description   |
+|-----------------------|--------|:-------------------------------------------|
+| plant_number          |        | Number of plants                           |
+
+The following sensors are currently available per plant:
+
+| name                  | Unit   | Description   |
+|-----------------------|--------|:-------------------------------------------|
+| last_update           |        | Time of latest data update.                |
+| plant_name            |        | Name of the plant (as set by user)         |
+| scientific_name       |        | Scientific name of the plant               |
+| plant_status          |        | FYTA-Status (number 1 to 5)                |
+| temperature_status    |        | FYTA-Status (number 1 to 5)                |
+| light_status          |        | FYTA-Status (number 1 to 5)                |
+| moisture_status       |        | FYTA-Status (number 1 to 5)                |
+| salinity_status       |        | FYTA-Status (number 1 to 5)                |
+| temperature           | °C     | Temperature measured by sensor             |
+| light                 | mol/d  | Light measured by sensor                   |
+| moisture              | %      | Moisture measured by sensor                |
+| salinity              | mS/cm  | Salinity measured by sensor                |
+| battery_level         | %      | Battery level of the sensor                |
+
+
+<div class='note'>
+The fyta integration is using the fyta_cli package to get the data from the Fyta server (over the public API).
+</div>
```

Comment:
This could be omitted

Reply:
Could or should?

Reply:
Should 😉 This is information that can be interesting for devs, but the docs focus on the normal user and for them it is of no added value.